cxx/2slice.hxx

```
@@ -357,6 +358,7 @@ public:
     }
 
     void _drop() noexcept;
+    BoxedSlice<T> clone() const noexcept;
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)